Code/GraphMol/ChemReactions/Reaction.cpp

```diff
@@ -646,4 +646,18 @@ void ChemicalReaction::removeUnmappedProductTemplates(
   res_productTemplates.clear();
 }
 
+void ChemicalReaction::removeAgentTemplates(
+    MOL_SPTR_VECT *targetVector)
+{
+  if(targetVector){
```

Comment:
Isn't this just

```
*targetVector = rxn.getAgents();
```

This also makes me think that it doesn't belong in this function since it is really easy to do outside the function.